src/index.html

```diff
@@ -7,8 +7,67 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;800&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="./images/logo.png" alt="logo__MOYO">
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-ul">
+          <li class="header__nav-li">
+            <a href="#"
+            class="header__nav-link is-active"
+            >Apple</a>
```

Comment:
```suggestion
            <a 
              href="#"
              class="header__nav-link is-active"
            >Apple</a>
```

Reply:
every attribute should be on new line(if you started moving it)
move attributes when you have them >= 2

Reply:
fix everywhere please